CHANGELOG.md

```diff
@@ -1,5 +1,104 @@
 # Change Log
 
+## [vX.X.X](https://github.com/eclipse-iceoryx/iceoryx/tree/v1.0.0) (unreleased)
+
+[Full Changelog](https://github.com/eclipse-iceoryx/iceoryx/compare/v1.0.1...master)
+
+**Features:**
+
+- Enhance posixCall[\#805](https://github.com/eclipse-iceoryx/iceoryx/issues/805)
+- New chunk available callback for the new C++[\#391](https://github.com/eclipse-iceoryx/iceoryx/issues/391)
+- Git Hooks on iceoryx[\#486](https://github.com/eclipse-iceoryx/iceoryx/issues/486)
+- static memory alternative for std::function[\#391](https://github.com/eclipse-iceoryx/iceoryx/issues/391)
+- Adding support for Helix QAC 2021.1[\#755](https://github.com/eclipse-iceoryx/iceoryx/issues/755) thanks to @toniglandy1
+- Axivion analysis on CI[\#409](https://github.com/eclipse-iceoryx/iceoryx/issues/409)
+
+**Bugfixes:**
+
+- Fix warnings for gcc-11.1[\#838](https://github.com/eclipse-iceoryx/iceoryx/issues/838)
+- Incremental builds with the build script are broken[\#821](https://github.com/eclipse-iceoryx/iceoryx/issues/821)
+- Compile failed because of missing <limits> for GCC 11[\#811](https://github.com/eclipse-iceoryx/iceoryx/issues/811) thanks to @homalozoa
+
+**Refactoring:**
+
+- Move all tests into an anonymous namespace[\#563](https://github.com/eclipse-iceoryx/iceoryx/issues/563)
+- Refactor smart_c to use contract by design and expected[\#418](https://github.com/eclipse-iceoryx/iceoryx/issues/418)
+- PoshRuntime Mock[\#449](https://github.com/eclipse-iceoryx/iceoryx/issues/449)
+- Clean-up Doxygen for all methods in headers[\#583](https://github.com/eclipse-iceoryx/iceoryx/issues/583)
+- Iox-#590 Rename utils to hoofs[\#790](https://github.com/eclipse-iceoryx/iceoryx/pull/790)
+- Iox-#590 plantuml in design documentation[\#787](https://github.com/eclipse-iceoryx/iceoryx/pull/787)
+- Refine quality levels[\#425](https://github.com/eclipse-iceoryx/iceoryx/issues/425)
+- Clean-up std::terminate usage[\#261](https://github.com/eclipse-iceoryx/iceoryx/issues/261)
+
+**API Breaking Changes:**
+
+Rename utils to hoofs:
+
+- in CMake you need now to find and link the package `iceoryx_hoofs` instead of `iceoryx_utils`
+
+```cmake
+# before
+find_package(iceoryx_utils REQUIRED)
+target_link_libraries(${target}
+    iceoryx_utils::iceoryx_utils)
+
+# after
+find_package(iceoryx_hoofs REQUIRED)
+target_link_libraries(${target}
+    iceoryx_hoofs::iceoryx_hoofs)
+```
+
+- the include paths for `iceoryx_utils` are now `iceoryx_hoofs`
+
+```cpp
+// before
+#include "iceoryx_utils/cxx/string.hpp"
+
+// after
+#include "iceoryx_hoofs/cxx/string.hpp"
+```
+
+Enhance posixCall to handle a common case were errnos are ignored just to suppress error logging:
```

Comment:
Oh, and the breaking change was the renaming of `smartC` to `posixCall` with the additional API changes. What you are mentioning here is the introduction of `suppressErrorMessagesForErrnos` which was not available in the old `smartC`